packages/plugin-moxie-token-transfer/src/utils/subgraph.ts

```diff
@@ -44,6 +45,9 @@ export async function getSubjectTokenDetailsBySubjectAddress(traceId: string, su
 
     while (retries < maxRetries) {
         try {
+            if (!PROTOCOL_SUBGRAPH_URL) {
```

Comment:
Would be better to move this at the very top of the function, so it doesn't go thru some of the logics above before landing here